benchmark/src/main/scala/keycloak/scenario/KeycloakScenarioBuilder.scala

```diff
@@ -312,11 +313,53 @@ class KeycloakScenarioBuilder {
         .header("Content-Type", "application/json")
         .body(StringBody("""{"id":"${createdRealmId}","realm":"${createdRealmId}","enabled": true}"""))
         .check(status.is(201))
-        .check(header("Location").notNull.saveAs("location")))
+        .check(header("Location").notNull.saveAs("realmLocation")))
+      .exec(session => (session.removeAll("createdRealmId")))
       .exitHereIfFailed
     this
   }
 
+  //Client Scopes
+  def createClientScopes(): KeycloakScenarioBuilder = {
+    chainBuilder = chainBuilder
+      .exec(_.set("createdClientScopeId", randomUUID()))
+      .exec(http("Create client scopes")
+        .post(ADMIN_ENDPOINT + "/client-scopes")
+        .header("Authorization", "Bearer ${token}")
+        .header("Content-Type", "application/json")
+        .body(StringBody(
+          """ {"attributes":{"display.on.consent.screen":"true","include.in.token.scope":"true"},
+            | "name":"${createdClientScopeId}","protocol":"openid-connect"} """.stripMargin))
+        .check(status.is(201))
+        .check(header("Location").notNull.saveAs("clientScopeLocation")))
+      .exec(session => (session.removeAll("createdClientScopeId")))
+      .exitHereIfFailed
+    this
+  }
+
+  def listClientScopes(): KeycloakScenarioBuilder = {
+    chainBuilder = chainBuilder
+      .exec(http("List client Scopes")
+        .get(ADMIN_ENDPOINT + "/client-scopes")
+        .header("Authorization", "Bearer ${token}")
+        .queryParam("max", 2)
```

Comment:
As per API spec and per implementatoin, no query parameters are supported. It will always return the full list of clients. To avoid confusion for those reviewing the scenario, please remove this query parameter.

Reply:
thanks for catching this, will remove this parameter